### ISSUE TITLE
MSE of NN prediction

### ISSUE THREAD

**Issue by aigerimhatake** [matheusAltomare/PINN#2]
Thank you for sharing your code!
I just want to know how to show the mse of the model? 
Visually it looks like NN performs perfectly, nevertheless I would like to see exact number of mse?

**Comment by matheusAltomare** [comment]
Good idea, in fact it was missing. But this is easy to do, in the final part of the notebook where the analytical solution is compared with that obtained via deep learning, just calculate the metric between these two using the analytical solution as y_true.

Soon I will make this addition to the code.
